### PR TITLE
Refresh filecoin-programs doc

### PR DESCRIPTION
**Pull request by jochasinga**
Working on adding Github, doc, and Slack visual badges and linking them to appropriate sources.

See also: [Adding Sparkle to Social Coding: An Empirical Study of
Repository Badges in the npm Ecosystem](https://cmustrudel.github.io/papers/icse18badges.pdf)

Screenshot 2024-05-16 at 1 52 20 PM

### CODE REVIEW

**Comment by jochasinga** [comment]
FYI there are many stale links scattered throughout this repo that aren't passing the check.

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 1]
```diff
@@ -10,40 +10,65 @@ Although it is possible to find your own data storage customers with valuable da
 
 ## Web3.storage
 
+[![GitHub Repo stars](https://img.shields.io/github/stars/web3-storage/web3.storage)](https://github.com/web3-storage/web3.storage)
+![GitHub last commit](https://img.shields.io/github/last-commit/web3-storage/web3.storage)
+[![Docs site](https://img.shields.io/badge/docs-web3.storage-blue)](https://web3.storage/docs/)
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:16]: Web3.storage is not deprecated...

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 2]
```diff
@@ -10,40 +10,65 @@ Although it is possible to find your own data storage customers with valuable da
...
 
-[Evergreen](https://evergreen.filecoin.io/) extends the Slingshot program by aiming to store the open datasets forever. Standard deals have a maximum duration of 540 days, which is not long enough for valuable, open datasets that might need to be stored forever. Evergreen uses a deal engine ([Spade](https://github.com/ribasushi/spade)) that automatically renews deals to extend the lifetime of the dataset on-chain.
+[Singularity](https://singularity.storage) is an end-to-end solution for onboarding datasets to Filecoin storage providers, supporting [PiB-scale data](https://stats.singularity.storage/). It offers modular compatibility with various data preparation and deal-making tools, allowing efficient processing from local or remote storage. Singularity integrates with over 40 storage solutions and introduces inline preparation, which links CAR files to their original data sources, preserving dataset hierarchies. It also supports content distribution and retrieval through multiple protocols and provides push and pull modes for deal making along with robust wallet management features.
 
-## Saturn
+## Evergreen
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:54]: evergreen moves to the deprecated section, as the https://evergreen.filecoin.io/ site is going away. Spade the tool is not going away, and should  get its own block in the live section.
Also should have an April 2024 closure date, along with the other deprecated items.
Also talk about deprecated programs in the past tense...

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 3]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
 Storing real client data means there will be expectations on retrievability of that data. The storage provider can provide the data to the client by keeping unsealed copies of the data and creating an access layer (web or other) for the client. Also Boost-http (see [https://lotus.filecoin.io](https://lotus.filecoin.io) ) provides a way to access data.
 
-### Filswan
+### Swan (Filswan)
+
+[![GitHub Org's stars](https://img.shields.io/github/stars/filswan)](https://github.com/filswan)
+[![Read the doc](https://img.shields.io/badge/docs-filswan.com-blue)](https://docs.filswan.com/)
+[![Join Slack](https://img.shields.io/badge/join-Slack-purple)](https://filecoinproject.slack.com/archives/C04LBAMBDPD)
+
+[Swan](https://www.filswan.com/homepage) is a leading provider of cross-chain cloud computing solutions. Its suite of tools revolutionizes how developers access resources across multiple chains.
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:86]: Are all these swan tools currently working? I see FS3 was last updated in 2021. If we're not sure they're working tooling, let's not include them. 
Also, let's just talk about what they actually do, practically, and avoid marketing language like 'revolutionizes' - no matter what language they use themselves.

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 4]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+
+A golang implementation for Swan Service provider.
+
+#### Multichain Storage
+
+[![Github repo stars](https://img.shields.io/github/stars/filswan/multi-chain-storage)](https://github.com/filswan/multi-chain-storage)
+![GitHub last commit](https://img.shields.io/github/last-commit/filswan/multi-chain-storage)
+[![Read the doc](https://img.shields.io/badge/docs-README-blue)](https://github.com/filswan/multi-chain-storage/README.md)
+
+[Multichain Storage](https://multichain.storage/home) is a smart-contract-based cross-chain storage gateway integrated with Oracle technology and the Filecoin networks. It accelerates the mass adoption of decentralized storage by bridging multiple blockchain networks.
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:114]: lower-case o in 'oracle' - they got it wrong.

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 5]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+
+#### FS3
+
+[![Github repo stars](https://img.shields.io/github/stars/filswan/fs3)](https://github.com/filswan/fs3)
+![GitHub last commit](https://img.shields.io/github/last-commit/filswan/fs3)
+[![Read the doc](https://img.shields.io/badge/docs-README-blue)](https://github.com/filswan/fs3/README.md)
+
+A high Performance, Kubernetes Native Object Storage.
+
+## Deprecated
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:124]: I can't remember what I said here - something like 'Former tools and programs' ?

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 6]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+
+Here is a comprehensive list of deprecated tools and projects.
+
+## Filecoin Tracker
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated_04/20/2024-red.svg)
+
+[Filecoin Tracker](https://filecoin-tracker.com/) was deprecated on April 20, 2024.
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:132]: I think we should put the url in brackets in the ## title, but there's no point in linking, as any traffic to those domains will be sent straight back here.
so like 
## Filecoin Tracker (filecoin-tracker.com) 

for all of the deprecated ones.

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 7]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+
+## Slingshot
+
+[![GitHub Repo stars](https://img.shields.io/github/stars/filecoin-project/slingshot)](https://github.com/filecoin-project/slingshot)
+![GitHub last commit](https://img.shields.io/github/last-commit/filecoin-project/slingshot)
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+[![Join Slack](https://img.shields.io/badge/join-Slack-purple)](https://filecoinproject.slack.com/archives/C01AZP8BKRQ)
+
+[Slingshot](https://slingshot.filecoin.io) was a program that united Data clients, Data preparers and storage providers in a community to onboard data and share replicas of publicly valuable [_Open Datasets_](https://datasets.filecoin.io). Rather than providing a web interface like Estuary, Slingshot was a program that provides a workflow and tools for onboarding of large open datasets. The Slingshot Deal Engine provided deals to registered and certified storage providers. The data was prepared and uploaded using a tool called Singularity. The program has [clear requirements of a storage provider](https://slingshot.filecoin.io/requirements#participating-as-a-storage-provider-sp) regarding their capabilities.
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:147]: This is good.
link 'Singularity' to the singularity github repo. and remove the last sentence.

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 8]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+
+See the below video for more information on Slingshot:
+
+{% embed url="https://www.youtube.com/watch?v=14sasiEThig" %}
+A overview of the Slingshot project from the Enterprise Storage Provider Accelerator (ESPA).
+{% endembed %}
+
+See also:
+
+- [Slingshot v2](https://v2.slingshot.filecoin.io/)
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:157]: can remove all these links, we'll be redirecting all to the main anchor.

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 9]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+- [Slingshot v3](https://v3.slingshot.filecoin.io/)
+- [Slingshot Recovery](https://recovery.slingshot.filecoin.io/)
+- https://evergreen.slingshot.filecoin.io/ ⚠️
+- https://dashboard.slingshot.filecoin.io/ ⚠️
+
+## Filecoin+
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:163]: remove this section. The Filecoin+ program is not being deprecated! (although some of the websites may be changing).

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 10]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+To enhance the Filecoin's utility, [Filecoin Plus](https://filplus.storage/) introduced a social mechanism that incentivizes participants to maximize useful storage. Data owners used DataCap to make deals with storage providers, who benefited from increased block rewards over time. The Filecoin Plus program involved various stakeholders, including root key holders, notaries, data owners, and storage providers, who collaborated to form a governed community.
+
+## Open Panda
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:169]: put the url in the heading...

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 11]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+
+[Open Panda](https://openpanda.io/) is a platform for data researchers, analysts, students and enthusiasts to interact with the largest open datasets in the world. Data available through our platform is stored on Filecoin, a decentralized storage network comprised of thousands of independent Storage Providers around the world.
+
+## Data Program
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+[Data Program](https://dataprograms.org) listed tools, products, and incentive programs designed to drive growth and make data storage on Filecoin more accessible. This list is currently more up-to-date.
+
+## Moonlanding
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:181]: put the url in the heading...

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 12]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+
+To enhance the Filecoin's utility, [Filecoin Plus](https://filplus.storage/) introduced a social mechanism that incentivizes participants to maximize useful storage. Data owners used DataCap to make deals with storage providers, who benefited from increased block rewards over time. The Filecoin Plus program involved various stakeholders, including root key holders, notaries, data owners, and storage providers, who collaborated to form a governed community.
+
+## Open Panda
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+[Open Panda](https://openpanda.io/) is a platform for data researchers, analysts, students and enthusiasts to interact with the largest open datasets in the world. Data available through our platform is stored on Filecoin, a decentralized storage network comprised of thousands of independent Storage Providers around the world.
+
+## Data Program
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:175]: * I'd actually make the header 

## Dataprograms.org 
and say it was discontinued in April 2024
* Remove the last sentence

File: storage-providers/filecoin-deals/filecoin-programs.md [hunk 13]
```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
...
+
+## Filecoin Dataset Explorer
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+[Filecoin Dataset Explorer](https://datasets.filecoin.io/) showcased data stored on the Filecoin network between 2020 and 2022, including telemetry, historical archives, Creative Commons media, entertainment archives, scientific research, and machine learning datasets. It highlighted Filecoin's capability to store large datasets redundantly, ensuring availability from multiple Storage Providers worldwide. Each dataset is identified by a unique content identifier (CID). The platform aimed to make diverse datasets accessible to users globally.
+
+See also: [Legacy Explorer](https://legacy.datasets.filecoin.io/)
+
+## Estuary
```
Comment (smagdali) [inline storage-providers/filecoin-deals/filecoin-programs.md:195]: ## Estuary (Estuary.tech)

*fix the tense. 
* Estuary was discontinued in July 2023, and the website shut down in April 2024. 
remove the note about documentation, we'll redirect that here as well.

**Review by smagdali — CHANGES_REQUESTED** [review]
getting there! comments below

File: storage-providers/filecoin-deals/filecoin-programs.md (hunk 4 above)
Comment (jochasinga) [reply storage-providers/filecoin-deals/filecoin-programs.md:114]: Let's just not expand their suite of tools here. I think it added too much overhead on keeping updated.

File: storage-providers/filecoin-deals/filecoin-programs.md (hunk 5 above)
Comment (jochasinga) [reply storage-providers/filecoin-deals/filecoin-programs.md:124]: Yes it's changed on the next push.

File: storage-providers/filecoin-deals/filecoin-programs.md (hunk 6 above)
Comment (jochasinga) [reply storage-providers/filecoin-deals/filecoin-programs.md:132]: I think the parenthesized URLs in the headings will make the doc look cluttered. Let's just remove them altogether or add them below as a footnote/reference.

**Comment by jochasinga** [comment]
> Also move CO2.storage to deprecated.

done

File: storage-providers/filecoin-deals/filecoin-programs.md (hunk 6 above)
Comment (jochasinga) [reply storage-providers/filecoin-deals/filecoin-programs.md:132]: I'll wait on this final touch.

**Comment by jochasinga** [comment]
> OK. looking good. A late-breaking change - move open panda back to the live list for now (it may get revived).

The website looks really legit. Hope it does.

**Pull request merged**